content/docs/learning-guide/tutorials/rendering/vegetation-bending-tutorial.md

```diff
@@ -59,44 +58,47 @@ Now we are ready to edit our shader to change how the engine renders our materia
 ### Render the material at an offset
 To start off, let's edit the vertex shader to render our shader ball at an offset.
 1. Open `{your-project-path}\Materials\Types\VegetationBending_ForwardPass.azsli`. Recall that `.azsli` files contain shader code. This file contains the vertex and pixel shader code for the *forward pass* of the vegetation bending material type.
-2. Find the function `VegetationBending_ForwardPassVS`.
-3. Towards the end of the function, right before `OUT.m_worldPosition = worldPosition.xyz;`, add: 
+1. Find the function `VegetationBending_ForwardPassVS`.
+1. Towards the end of the function, right before `OUT.m_worldPosition = worldPosition.xyz;`, add: 
    
    ```hlsl
-   worldPosition.x += 5;
+   worldPosition.x += 5.0;
    ```
-   This will adjust the position in the positive x direction by 5 units. You may wonder why we are editing `worldPosition` instead of `m_position`; `m_position` is the position of this vertex relative to the origin of the model, whereas `m_worldPosition` is the position of this vertex relative to the origin of the level (or world). Try out editing the other dimensions and `m_position` and `worldPosition` and see what they do!
-4. Make sure the **Editor** is open, if it is not already open.
-5. Save your file with **CTRL-S** and the **Asset Processor** should automatically detect changes and process the file. You can open the **Asset Processor** and check when the file is done processing. 
+   This will adjust the position in the positive x direction by `5` units. You may wonder why we are editing `worldPosition` instead of `m_position`; `m_position` is the position of this vertex relative to the origin of the model, whereas `m_worldPosition` is the position of this vertex relative to the origin of the level (or world). Try out editing the other dimensions and `m_position` and `worldPosition` and see what they do!
+1. Make sure the **Editor** is open, if it is not already open.
+1. Save your file with **CTRL-S** and the **Asset Processor** should automatically detect changes and process the file. You can open the **Asset Processor** and check when the file is done processing. 
    {{< note >}}
    If you can't find the **Asset Processor**, click the arrow on the bottom right of your taskbar, and then click on the icon of two arrows in a circle.
    {{< /note >}}
-6. Once the **Asset Processor** is done processing the changes, you should see in the **Editor** that your material looks different!
+1. Once the **Asset Processor** is done processing the changes, you should see in the **Editor** that your material looks different!
+   {{< note >}}
+   The Editor viewport will not update unless the Editor window is in focus.
```

Comment:
This is no longer true, @gadams3 fixed it to reload even when not in focus.

Reply:
Thanks, fixed